Sources/OpenTelemetrySdk/Trace/TraceConstants.swift

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+
+public enum TraceConstants: String {
```

Comment:
so this is a public class on SDK? Is it assumed that libraries that instrument code will take dependency on SDK rather than API?

Reply:
maybe not public on not on SDK. Also, would it be better to create wrappers for errors. Is it unnetural to operate with the word "exception" on swift?

Reply:
> so this is a public class on SDK? Is it assumed that libraries that instrument code will take dependency on SDK rather than API? 

That's correct.

> maybe not public on not on SDK. Also, would it be better to create wrappers for errors. Is it unnatural to operate with the word "exception" on swift?

I'm following the established pattern in OpenTelemetrySDK/Resources/ResourcesConstant.swift per public and formatting
These specific strings are pulled from the exception spec : https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/exceptions.md
These constants are intended to be used in that context, so I think leaving it using a generic term "exceptions" should be ok.

I'd like to add a recordException() to span object, but this is good first step.